src/snowflake/snowpark/files.py

```diff
@@ -93,58 +93,119 @@ def close(self) -> None:
         return
 
     def detach(self) -> None:
+        """
+        Not yet supported in UDF and Stored Procedures.
+
+        See https://docs.python.org/3/library/io.html#io.BufferedIOBase.detach
+        """
```

Comment:
Based on SfStream implementation this actually returns `UnsupportedOperation("Detaching stream from file is unsupported")`

Reply:
Doesn't that match with Not yet supported in UDF and Stored procedures?